src/rules/function-name-case/README.md

```diff
@@ -0,0 +1,97 @@
+# function-name-case
+
+Specify lowercase or uppercase for functions.
+
+```css
+      a { width: calc(5% - 10em); }
```

Comment:
Can we have this flush to the left please? We only align the arrow with the final letter in the comment when the comment is shorter than the code snippet.